dist/checkbox_input.sh

```diff
@@ -154,7 +154,7 @@ select_indices() {
 
 
 on_checkbox_input_up() {
-  remove_checkbox_instructions
+  #remove_checkbox_instructions
```

Comment:
Thanks for attempting to fix the issue! Unfortunately, I don't really get why `remove_checkbox_instructions` should be commented out, it changes the behavior of the instructions as I originally intended, since I intended for it to disappear on the first keystroke.  It doesn't also really solve the root cause of the problem, which is that the terminal screen is not re-rendered upon terminal resize.